tests/playbooks/file_remote.yaml

```diff
@@ -25,6 +25,8 @@
       pulp_file_remote:
         name: test_file_remote
         url: "{{ pulp_fixtures_url }}/file/PULP_MANIFEST"
+        proxy_url: "{{ pulp_proxy_url | default(omit) }}"
+        tls_validation: "{{ pulp_tls_validation | default(omit) }}"
```

Comment:
Now comes the hard part. In this file we want to test file_remotes. So it would be helpful to add real values here (not `omit`). And that should most definitely lead to the need to rerecord this test.
(This is probably the right time to document that procedure...)
In short, you need to configure your pulplift instance in `vars/server.yaml` and call `make record_file_remote` (in a virtualenv?!).

Reply:
See general comment: https://github.com/mdellweg/ansible_modules_pulp/pull/6#issuecomment-561291464